ci-operator/config/openstack-k8s-operators/repo-setup/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
+- rlandy
+- raukadah
+- marios
+- cedric
```

Comment:
Based on https://github.com/openshift/release/pull/39063#issuecomment-1538184667, He needs to join openshift org. For timebeing , we can drop cedric from here.